Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN npm ci
 
 COPY . .
 ENV APP_BUILD_HASH=${BUILD_HASH}
+ENV NODE_OPTIONS="--max_old_space_size=8192"
```

Comment:
Should be removed: https://github.com/open-webui/open-webui/pull/5639

Reply:
Done